internal/resourcepolicies/resource_policies.go

```diff
@@ -122,16 +135,27 @@ func (p *Policies) match(res *structuredVolume) *Action {
 	return nil
 }
 
-func (p *Policies) GetMatchAction(res any) (*Action, error) {
+func (p *Policies) GetMatchAction(volumeRes any, pvcRes any) (*Action, error) {
```

Comment:
The resource policy docs suggest that resource policies are defined only for volumes now but may be expanded in the future, so I think the param names should probably remain generic. Keep the primary one as `res` like before, and maybe `secondaryRes` for the pvc -- i.e. a second object that's only used in relation to the  first